project.clj

```diff
@@ -75,6 +76,12 @@
    [com.mattbertolini/liquibase-slf4j "2.0.0"]                        ; Java Migrations lib logging. We don't actually use this AFAIK (?)
    [com.mchange/c3p0 "0.9.5.3"]                                       ; connection pooling library
    [com.taoensso/nippy "2.14.0"]                                      ; Fast serialization (i.e., GZIP) library for Clojure
+   [commons-codec/commons-codec "1.12"]                               ; Apache Commons -- useful codec util fns
```

Comment:
these were already pulled in by various other dependencies, I've made them explicit so we can use some of these functions more readily and not have to reimplement the wheel ourselves